src/sentry/api/helpers/group_index/update.py

```diff
@@ -250,7 +250,7 @@ def update_groups(
                 .order_by("-sort")[0]
             )
             activity_type = Activity.SET_RESOLVED_IN_RELEASE
-            activity_data: MutableMapping[str, Optional[Any]] = {
+            activity_data: MutableMapping[str, Any | None] = {
```

Comment:
@mgaeta were these changes caused by linting? Or manual changes?

Reply:
This change was automatically applied by the git-commit hook `pyupgrade`.